DataFormats/Provenance/src/Hash.cc

```diff
@@ -76,7 +76,7 @@ namespace edm {
       value_type temp1(hash);
       fixup_(temp1);
       cms::MD5Result temp;
-      copy_all(temp1, temp.bytes);
+      copy_all(temp1, temp.bytes.begin());
```

Comment:
```suggestion
      copy_all(temp1, temp.bytes.data());
```
?
`begin()` supposedly returns an iterator, is it guaranteed to be a pointer ?

Reply:
actually no, I guess `copy_all` does take an iterator...